### DISCUSSION TITLE
How to hide the apisix information of the server in the response header?

### DISCUSSION THREAD

**Discussion by solo-xwz** [apache/apisix#5697] [Q&A]
The kong gateway can be set not to display, and Apisix does not know if there is a parameter to hide it?

**Comment by tokers** [comment]
Do you mean hide the `Server` header or hiding the version?

**Reply by tzssangglass** [reply]
see: https://github.com/apache/apisix/blob/e90e3b7aa1d1789999b3ac740ebf6fcf887ff951/conf/config-default.yaml#L47

we can set `enable_server_tokens` as `false` in `config.yaml`.

**Reply by solo-xwz** [reply]
> Do you mean hide the `Server` header or hiding the version?

 I mean hide the Server header，or show the default nginx

**Reply by tzssangglass** [reply]
we can not do this, refer to: https://github.com/apache/apisix/issues/5353#issuecomment-953537691

**Reply by solo-xwz** [reply]
> 

So, where is the source code for adding apisix information in Server, thank you

**Reply by tzssangglass** [reply]
https://github.com/apache/apisix/blob/3d5f554c66903e0dc888303e92d9adf122863dbf/apisix/init.lua#L599
